src/Text/Regex/Base/Context.hs

```diff
@@ -11,29 +11,29 @@ Stability   :  experimental
 Portability :  non-portable (MPTC+FD)
 
 This is a module of instances of 'RegexContext' (defined in
-Text.Regex.Base.RegexLike).  Nothing else is exported.  This is
-usually imported via the Text.Regex.Base convenience package which
-itself is re-exported from newer Text.Regex.XXX modules provided by
-the different regex-xxx backends.
+''Text.Regex.Base.RegexLike'').  Nothing else is exported.  This is
```

Comment:
Why double single quotes instead of single double quotes? ;)

Reply:
Ah, thanks.  Will fix.